client/src/pages/layout/Navbar.js

```diff
@@ -1,8 +1,12 @@
 import React from "react";
 import { Link } from "react-router-dom";
 import { Navbar as NavbarBootstrap, NavDropdown, Nav } from "react-bootstrap";
+import { useAuth } from "./../../redux/hooks";
+
 import "./Navbar.css";
 const Navbar = () => {
+  const { isAuthenticated, logoutUser, user } = useAuth();
+
   return (
     <NavbarBootstrap className="navbar-custom" bg="transparent" expand="lg">
```

Comment:
I am thinking that maybe you could have two navbars: one for authenticated user and another for unauthenticated. I think It would be easier to add or remove new options to the navbar.